samples/kubernetes/end2end/docs/03-wls-operator.md

```diff
@@ -3,12 +3,18 @@ We depend on the [WLS Kubernetes Operator](https://github.com/oracle/weblogic-ku
 
 In this example, we provide the steps to install the 2.1 release of the WLS Kubernetes Operator.
 
+### Pulling the Images
+We need two images, one is the WLS Kubernetes Operator image from [Docker Hub](https://hub.docker.com) and the other is the WebLogic image from [Oracle Container Registry](https://container-registry.oracle.com).  
```

Comment:
We need two images, the WebLogic Kubernetes Operator image from [Docker Hub](https://hub.docker.com) and the  WebLogic Server image from [Oracle Container Registry](https://container-registry.oracle.com).